bindings/capacitorjs/src/lib/AccountManager.ts

```diff
@@ -0,0 +1,432 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { MessageHandler } from './MessageHandler';
+import { Account } from './Account';
+
+import type {
+    AccountId,
+    Auth,
+    EventType,
+    AccountManagerOptions,
+    CreateAccountPayload,
+    NodeInfoWrapper,
+    ClientOptions,
+    AccountSyncOptions,
+    WalletEvent,
+    LedgerNanoStatus,
+    Event,
+    EventId,
+    Node,
+    EventStatus,
+    GenerateAddressOptions,
+} from '../types';
+
+/** The AccountManager class. */
+export async function AccountManager(options: AccountManagerOptions) {
```

Comment:
Maybe add a return type here? Also a comment that explain why this is a function instead of a class would be nice.

Reply:
AFAIK TypeScript can infer the return type of `AccountManager` based on the types of the variables and functions inside the `AccountManager` factory function, without the need for explicit type annotations.
I think it's a good practice to let TypeScript's type inference system work for us as much as possible, as it can help catch errors early and improve the readability and maintainability of the code.

I'll add a detailed description of the why I've changed the classes by factory functions at the README.md file.